### PR TITLE
Suggest project structure

### PR DESCRIPTION
**Pull request by diedsmiling**
### Suggestion

This is a suggestion for default project structure. Having these in master ASAP would makes thing easier. 

## Loom packages

I see 2 packages that are minimum required for a loom application:

1) **Registry** - for centralised loom modules store.
2) **Loom** - Loom runtime itself.

Why to separate them in two different packages? Because they are totally different enteties. **Registry** is meant to be a web server / data storage. **Loom** is a React runtime module. 

This PR is not suggesting how the packages will be managed (Lerna, yarn workspaces etc.). Just the default structure.

### CODE REVIEW

File: packages/loom/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "loom-composer",
```
Comment (omarahm) [inline packages/loom/package.json]:
```suggestion
  "name": "@loomjs/loom",
```

File: packages/loom/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "version": "0.0.1",
+  "description": "Runtime for composing loom applications",
+  "main": "lib/index.js",
+  "repository": "https://github.com/loomjs/loom/tree/master/packages/loom-composer",
+  "author": "Alexandr Lazarev <diedsmiling@gmail.com>, Omar Ahmed <omar@omar.de>",
```
Comment (omarahm) [inline packages/loom/package.json]: Loom.js Contributors <team@loomjs.dev> ?

File: packages/registry/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+### Registry
```
Comment (omarahm) [inline packages/registry/README.md]: Can probably wait, but I really how you called it "Modules Store" in your PR, maybe we should call it that? What do you think? Probably need input from more team members

File: packages/registry/package.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "loom-registry",
```
Comment (omarahm) [inline packages/registry/package.json]:
```suggestion
  "name": "@loomjs/registry",
```

File: packages/registry/package.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+  "version": "0.0.1",
+  "description": "Centralised store for loom modules",
+  "main": "lib/index.js",
+  "repository": "https://github.com/loomjs/loom/tree/master/packages/loom-registry",
+  "author": "Alexandr Lazarev <diedsmiling@gmail.com>, Omar Ahmed <omar@omar.de>",
```
Comment (omarahm) [inline packages/registry/package.json]: Same as above, I vote for using the more generic Loom Contributors as the author

File: packages/loom/package.json (hunk 1 above)
Comment (diedsmiling) [reply packages/loom/package.json]: Good point! Thanks!

File: packages/registry/README.md (hunk 3 above)
Comment (diedsmiling) [reply packages/registry/README.md]: I also though about calling it something like "Modules state". "Store" might be confusing, people might think about physical storage of the modules (artifacts / js bundles).

Comment (omarahm) [reply packages/registry/README.md]: True that, let's postpone the decision for now then

Comment (diedsmiling) [reply packages/registry/README.md]: Will move it to issue for further discussion.

**Pull request merged**